src/app.js

```diff
@@ -1 +1,40 @@
-// write code here
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function moveFile() {
+  const [moveFrom, moveTo] = process.argv.slice(2);
```

Comment:
It would make more sense to have args parsing outside of the `moveFile` function and pass them down as parameters